test/problem2_spec.rb

```diff
@@ -0,0 +1,42 @@
+
+require 'rspec'
+require '../src/problem2'
+
+describe Problem2 do
+  describe '.get_largest_two_of' do
+    context 'when is null array' do
+      it 'returns nil' do
+        expect(Problem2.get_largest_two_of(nil)).to eql(nil)
```

Comment:
You can leave out `get` from method names https://github.com/bbatsov/ruby-style-guide#accessor_mutator_method_names